debian/extras/linuxcnc.files

```diff
@@ -9,10 +9,10 @@ usr/share/icons/hicolor/scalable/apps/linuxcnc-logo.svg
 usr/share/desktop-directories/linuxcnc-cnc.directory
 usr/share/desktop-directories/linuxcnc-doc.directory
 usr/share/desktop-directories/linuxcnc-ref.directory
-usr/share/icons/hicolor/scalable/apps/linuxcnc-en.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-es.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-fr.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-vi.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-cn.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_en.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_es.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_fr.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_vi.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_cn.svg
```

Comment:
These renames require that you also update the Icon filenames in the .desktop files in `debian/extras/usr/share/applications/`.